### ISSUE TITLE
🐛 [Bug] Torch-TensorRT does not support directly setting input to output

### ISSUE THREAD

**Issue by cyfwry** [NVIDIA/Torch-TensorRT#812]
##  Bug Description

Sometimes, due to fallback segmentation, or output unpacking, the input will be directly used as the output, which is not supported by TensorRT.

```
DEBUG: [Torch-TensorRT TorchScript Conversion Context] - graph(%x : Tensor):
  %2 : int[] = prim::Constant[value=[0, 0]]()
  %3 : int[] = prim::Constant[value=[1, 1]]()
  %4 : int = prim::Constant[value=1](), scope: __module.conv # /usr/local/lib64/python3.6/site-packages/torch/nn/modules/conv.py:443:0
  %5 : bool = prim::Constant[value=0](), scope: __module.conv # /usr/local/lib64/python3.6/site-packages/torch/nn/modules/conv.py:443:0
  %6 : bool = prim::Constant[value=1](), scope: __module.conv # /usr/local/lib64/python3.6/site-packages/torch/nn/modules/conv.py:443:0
  %self.conv.weight : Float(3, 3, 3, 3, strides=[27, 9, 3, 1], requires_grad=0, device=cuda:0) = prim::Constant[value=<Tensor>]()
  %self.conv.bias : Float(3, strides=[1], requires_grad=0, device=cuda:0) = prim::Constant[value= 0.0282  0.1217 -0.0183 [ CUDAFloatType{3} ]]()
  %9 : Tensor = aten::_convolution(%x, %self.conv.weight, %self.conv.bias, %3, %2, %3, %5, %2, %4, %5, %5, %6, %6), scope: __module.conv # /usr/local/lib64/python3.6/site-packages/torch/nn/modules/conv.py:443:0
  return (%x, %9)

...

DEBUG: [Torch-TensorRT] - Weights: [3, 3, 3, 3]
    Data Type: Float32
    Number of input maps: 3
    Number of output maps: 3
    Element shape: [3,3]
DEBUG: [Torch-TensorRT] - Input dims: [1, 3, 20, 20]
DEBUG: [Torch-TensorRT] - Weights: Weights: [3, 3, 3, 3]
    Data Type: Float32
    Number of input maps: 3
    Number of output maps: 3
    Element shape: [3,3]
DEBUG: [Torch-TensorRT] - stride: [1, 1]
DEBUG: [Torch-TensorRT] - padding: [0, 0]
DEBUG: [Torch-TensorRT] - dilation: [1, 1]
DEBUG: [Torch-TensorRT] - out_padding: [0, 0]
DEBUG: [Torch-TensorRT] - groups: 1
DEBUG: [Torch-TensorRT] - Output tensor shape: [1, 3, 18, 18]
ERROR: [Torch-TensorRT TorchScript Conversion Context] - 4: Tensor output_0 cannot be both input and output
INFO: [Torch-TensorRT TorchScript Conversion Context] - Marking Output x named output_0 in engine (ctx.MarkOutput)
INFO: [Torch-TensorRT TorchScript Conversion Context] - Marking Output 9 named output_1 in engine (ctx.MarkOutput)

```

## To Reproduce

Just run:

```
import torch
import torch.nn as nn
import torch_tensorrt as tt

class Model(nn.Module):
    def __init__(self):
        super(Model,self).__init__()
        self.conv=nn.Conv2d(3,3,3)

    def forward(self,x):
        x1=self.conv(x)
        return x,x1

a=Model().cuda().eval()
b=torch.jit.trace(a,torch.ones([1,3,20,20]).cuda())

compile_settings = {}
compile_settings["inputs"] = [tt.Input(shape = [1,3,20,20])]

r=tt.compile(b,**compile_settings)
r(torch.ones([1,3,224,224]).cuda())

```

## Expected behavior

The model compiles and infers correctly.

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - Torch-TensorRT Version : 1.0.0
 - PyTorch Version: 1.10.0
 - CPU Architecture: Intel(R) Xeon(R) Platinum 8352Y CPU @ 2.20GHz
 - OS (e.g., Linux): CentOS 7
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source): pip
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version: 3.6.8
 - CUDA version: 11.4
 - GPU models and configuration: A30
 - Any other relevant information:

## Additional context

I have fixed this bug locally. After confirming the existence of this bug, I will open a PR to fix it.

**Comment by cyfwry** [comment]
您好，您发给我的邮件已收到！

**Comment by ncomly-nvidia** [comment]
Fixed in #759
